twisted_fix/internet/inotify.py

```diff
@@ -0,0 +1,428 @@
+# -*- test-case-name: twisted.internet.test.test_inotify -*-
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
```

Comment:
Could you just create a class that inherits the actual twisted inotify, then overwrites the functions that you need changing?